playbooks/portals-setup-following.yml

```diff
@@ -105,10 +105,10 @@
       include_tasks: tasks/portal-get-log-filename-prefix.yml
 
     - name: Include getting Ansible repo branch and commit
-      include_tasks: tasks/playbook-version-get.yml
+      include_tasks: tasks/portal-version-get.yml
 
     - name: Include logging Ansible version
-      include_tasks: tasks/playbook-logs-ansible-log-version.yml
```

Comment:
renamed because `playbook` is a key word and caused errors on that tasks file.

Reply:
similar here